### ISSUE TITLE
Upload wheels to PyPI

### ISSUE THREAD

**Issue by johnthagen** [django-dbbackup/django-dbbackup#382]
Wheels (`.whl`) for this package are [currently missing from PyPI](https://pypi.org/project/django-dbbackup/3.3.0/#files). Could wheels be uploaded for the current and future releases?

Read more about the [advantages of wheels](https://pythonwheels.com/) to understand why generating wheel distributions are important.

To create a wheel along with source distribution:

```bash
(venv) $ pip install --upgrade pip setuptools wheel
(venv) $ python setup.py sdist bdist_wheel

# See dist/*.whl
```

To upload wheels:

```bash
(venv) $ pip install twine
(venv) $ twine upload dist/*
````

**Comment by u3556490** [comment]
Talking about PyPI, I noticed that the package code on PyPI appears to be outdated. I suppose it would be most suitable to report it here as a comment to another PyPI-related issue. The details of this problem is as follows:

When one downloads the package files from PyPI ([here](https://pypi.org/project/django-dbbackup/#files)) and extracts them, it appears that the Django app code is still in `3.1.5` (e.g. `dbbackup/management/commands/dbbackup.py` still lacks the `-x` option). Note that `dbbackup\__init__.py` claims otherwise: it says the version is` 3.3.0`. When installed via pip, pip would report the version as `3.3.0`, but when the app gets registered to a Django project, the version is reported to be `3.1.5` when `python manage.py dbbackup --version` is run.

I suppose the community would be very grateful if the package files on PyPI could be updated soon.

**Comment by johnthagen** [comment]
@jonathan-s Is this something you are able to look into? Not sure who exactly has PyPI permissions.

**Comment by johnthagen** [comment]
Closed in #412 

Wheels exist in `4.0.0b0`: https://pypi.org/project/django-dbbackup/4.0.0b0/#files
